### PR TITLE
Consolidate Information into Easier Parsed File

### PR DESCRIPTION
**Pull request by blackstar257**
Wanted to see if it would work.It does work fine on the rendered version through GitHub Pages just doesn't look good when viewing README.md directly next to the code. I thought this would be beneficial for future projects that want to use this as a source of data. They could just pull the latests .CSV file instead of having to parse the HTML rendered code.

Looking for feedback on the README.md to make it look better 
Example:
https://blackstar257.github.io/caprivacy/

README.md showing the un-rendered template tags:
https://github.com/blackstar257/caprivacy

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,123 +12,9 @@ Under AB-375, California residents have the right to:
 The following is an alphabetical ordered list of companies and URLs to perform a Data Subject Access Request (DSAR) and exercise your rights under this law:
 
 # Directory
```
Comment (caprivacy) [inline README.md]: Maybe make this a link or similar to the rendered version? [Directory](https://caprivacy.me)

**Review by caprivacy — CHANGES_REQUESTED** [review]
I like this! Thankfully it renders on caprivacy.me so I've suggested adding a link on the Directory.

Would folks then need to send updated CSV entries to add a link on a pull?

**Comment by blackstar257** [comment]
Yeah they would just do pull requests to modify the CSV file instead simplifying it

**Comment by blackstar257** [comment]
I tried to do it the simplest way without losing any of your information.

**Comment by blackstar257** [comment]
This way it looks good on Github side and on your website.

**Comment by blackstar257** [comment]
Let me know if there is anything else you think i should change @caprivacy

**Comment by blackstar257** [comment]
Also it then becomes easy to just add columns for more information for the company. Just would need to add columns to the CSV and update the index.md rendering to display how you like.

**Comment by liamengland1** [comment]
What if there are two urls? Example xfinity

**Review by caprivacy — APPROVED** [review]
That works too!

**Pull request merged**